ingestion/src/metadata/ingestion/source/dashboard/metabase/client.py

```diff
@@ -72,29 +78,28 @@ def get_dashboards_list(self) -> Optional[List[dict]]:
         """
         resp_dashboards = self.req_get("/api/dashboard")
         if resp_dashboards.status_code == 200:
-            return resp_dashboards.json()
-        return []
-
-    def get_dashboard_name(self, dashboard: dict) -> str:
-        """
-        Get Dashboard Name
-        """
-        return dashboard["name"]
+            return MetabaseDashboardList(dashboards=resp_dashboards.json())
+        logger.warning(f"Failed to fetch the dashboards: {resp_dashboards.text}")
+        return MetabaseDashboardList(dashboards=[])
```

Comment:
I believe instead of returning this return `None` and then handle the same wherever this is called

Reply:
if we are looping it in the next topology, i believe keeping an empty list should work as expected

Reply:
if we are looping it in the next topology, i believe keeping an empty list should work as expected